docs/root/operations/cli.rst

```diff
@@ -67,6 +67,12 @@ following are the command line options that Envoy supports.
   :option:`--restart-epoch` is non-zero. Instead, for subsequent hot restarts, set
   :option:`--base-id` option with the selected base ID. See :option:`--base-id-path`.
 
+.. option:: --skip-hot-restart-on-no-parent
+
+  *(optional)* In conjunction with :option:`--restart-epoch`, this flag allows for a failing hot
+  restart to fall back to normal startup behavior. When this flag is false, if the parent instance
+  was terminated, the child instance will also terminate during startup.
```

Comment:
maybe clarify that this is only available when the hot restart initialize stage. If the parent was terminated after hot restart, this won't help.

Reply:
Done.